blackbox/docs/config/cluster.rst

```diff
@@ -276,15 +276,16 @@ nodes of the cluster:
   to reallocate shards in order to ensure the minimum data availability set
   with ``min_availability``.
 
+  .. NOTE::
+     Currently, this setting is being ignored - setting the value to `false`
```

Comment:
I'd have gone as far as changing the text above that explains what `false` does.

From a user perspective:

First you're told that setting this to false causes the graceful stop to fail if the cluster would need to reallocate shards and then you're told that what you just read isn't true.

Sorry for the nitpicking. But I think we've to make it a habit to think about the reading flow of users.

Reply:
I did think about that but I also thought that I should keep the intended behavior (so I can better explain the reason we deprecate it). I also wanted to highlight that there is no effect - that's the reason I  added the ``NOTE``, so users don't miss it.

In any case, I get your point, I'm fine with doing the change you propose.